binderhub/static/index.js

```diff
@@ -164,48 +173,20 @@ $(function(){
     // setup badge dropdown
     updateUrl();
 
-    // start the badge invisible
-    $('#badge-snippet').hide();
-    $('#launch-buttons').on('hidden.bs.dropdown', function () {
-      // hide the badge snippet when dismissing the dropdown
-      $('#badge-snippet').hide();
-    });
-    $("#url-or-file-btn").find("a").click(function (evt) {
-      $("#url-or-file-selected").text($(this).text());
-      updatePathText();
-    })
-    updatePathText();
-
-    // prevent badge text dropdown menu from disappearing on click
-    $('.dropdown-menu#badge-text li').click(function(e) {
-          e.stopPropagation();
-    });
-
-    $('#markdown-badge-toggle').on('click', function () {
-      var url = updateUrl();
-      $('#badge-snippet')
-        .show()
-        .text(markdownBadge(url));
-      return false;
+    $('#repository').on('keyup paste', function () {
+      updateUrlDiv()
     });
```

Comment:
Nitpicking  and this is readable enough, but I believe this is equivalent to
 `$('#repository').on('keyup paste', updateUrlDiv);`
Same on the 3 below.